infrastructure-scripts/claude_start.sh

```
@@ -0,0 +1,1633 @@
+    if [ ! -d "$SCRIPT_DIR/orchestration" ]; then
+        return 1
+    fi
+
+    # Check if start script exists
+    if [ ! -f "$SCRIPT_DIR/orchestration/start_system.sh" ]; then
+        return 1
+    fi
+
+    # Start orchestration silently in background
+    (
+        cd "$SCRIPT_DIR/orchestration"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)